Python/kemoJourneyClothGate.py

```diff
@@ -155,20 +155,19 @@ def OnNotify(self,state,id,events):
             PtDebugPrint ("You're likely to be eaten by a grue...")
             vault = ptVault()
             entry = vault.findChronicleEntry("JourneyClothProgress")
-            if entry is not None:
```

Comment:
`ptVault.findChronicleEntry()` can return `None`. I'm not sure why the False case is unhandled though - maybe another script creates the chronicle before this happens. I guess this is ok...

Reply:
Then was [this comment](https://github.com/H-uru/moul-scripts/pull/120#discussion_r393145437) by mistake?

Reply:
Then was [this comment](https://github.com/H-uru/moul-scripts/pull/120#discussion_r393145437) by mistake?

Reply:
indeed